modules/services/analytics/src/main/java/au/com/shiftyjelly/pocketcasts/analytics/AnalyticsEvent.kt

```diff
@@ -296,4 +296,14 @@ enum class AnalyticsEvent(val key: String) {
     EPISODE_DETAIL_DISMISSED("episode_detail_dismissed"),
     EPISODE_DETAIL_PODCAST_NAME_TAPPED("episode_detail_podcast_name_tapped"),
     EPISODE_DETAIL_SHOW_NOTES_LINK_TAPPED("episode_detail_show_notes_link_tapped"),
+
+    /* End of Year */
+    END_OF_YEAR_MODAL_SHOWN("end_of_year_modal_shown"),
+    END_OF_YEAR_STORIES_SHOWN("end_of_year_stories_shown"),
+    END_OF_YEAR_STORIES_DISMISSED("end_of_year_stories_dismissed"),
+    END_OF_YEAR_STORIES_FAILED_TO_LOAD("end_of_year_stories_failed_to_load"),
+    END_OF_YEAR_STORY_REPLAY_BUTTON_TAPPED("end_of_year_story_replay_button_tapped"),
+    END_OF_YEAR_STORY_SHOWN("end_of_year_story_shown"),
```

Comment:
You might see an issue with this event because of the pause/ start issue that I'm looking into. I should have a PR with a fix by tomorrow.